site/src/components/gradient-card.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Gradient card
+description: The gradient card component in the Atlas Design System
+template: standard
+classType: Component
+classPrefixes:
+  - gradient-card
+---
+
+# Gradient card
+
+The `.gradient-card` class contains a predefined gradient border, default border width and default border radius. The inner `.gradient-card-content` element has a default background color and padding but can be modified by Atomic classes.
+
+```html
+<div class="gradient-card">
+	<div class="gradient-card-content">
+		<p>Gradient card default</p>
+	</div>
+</div>
+```
+
+```html
+<div class="gradient-card">
+	<div class="gradient-card-content background-color-body-medium padding-md">
+		<p>Gradient card with atomics</p>
+	</div>
+</div>
+```
+
+## Usage
+
+Gradient cards are used to bring attention to a section or for banners. Here is a common use case.
```

Comment:
I'm trying to recall how these will be used and wondering if "banners" is the right word here, as we have both a banner and a notification component. Please review the usage of the two and see which one is more appropriate. If it is indeed for banners, then here's what it would look like. 
![image](https://github.com/microsoft/atlas-design/assets/94572161/7b41adfb-3b33-4c8a-bd4a-ab1d255b7fb1)

Reply:
I'll change it to 'notifications'

Reply:
Should it have a `notification` component nested? Please check out relevant comment down below I just posted.